src/main/jte/layout.jte

```diff
@@ -120,6 +120,10 @@
                             <li><a href="/admin/rooms"><i class="fa-solid fa-house"></i> ${context.localize("rooms")}</a></li>
                         @endif
 
+                        @if(layout.getLogin().kind() == LoginStateModel.Kind.ADMIN && layout.getRoom() != null)
+                            <li><a href="/admin/opening-hours/${layout.getRoom().getId()}"><i class="fa-solid fa-list"></i> ${context.localize("administration.opening_hours")}</a></li>
+                        @endif
+
```

Comment:
We already have an `if` block for admins. I think adding a new block under that instead of this separate block would be cleaner